pages/blog/march-2021-at-asyncapi.md

```diff
@@ -43,7 +43,7 @@ Let's scale this party up!
 We did it. AsyncAPI initiative joined Linux Foundation (LF). No more excuses for you not to join us. You're all welcome.
 
 I don't want to write about this too much as everything was described in different articles about us joining LF:
-- [AsyncAPI joins Linux Foundation](http://localhost:8080/blog/asyncapi-joins-linux-foundation)
+- [AsyncAPI joins Linux Foundation](https://www.asyncapi.com/blog/asyncapi-joins-linux-foundation)
```

Comment:
Regarding this change, I think it's necessary to fix the broken links on the website. And surprisingly, I got this :smile:.